src/app/components/harvest/pages/list/list.component.ts

```diff
@@ -92,6 +92,10 @@ class ListComponent extends PageComponent implements OnInit {
     return model;
   }
 
+  public harvestCreator(model: Harvest): User {
+    return model.creator;
+  }
+
```

Comment:
does angular change detection still penalize methods?

Regardless seems like a little too much ceremony to have a whole function just to access one property

Reply:
I agree. I initially tried to use the `model.creator` directly in the template, but it didn't seem to work because change detection wouldn't fire when the associated model would change from an `UnresolvedModel` to a resolved model.

Using a wrapper function like this made change detection work as expected and updated the templated model when the associated model changed.

If I can't use the `model.creator` directly in the template, what would you recommend for the best path forward? Fix the underlying issue in the association decorator to make change detection fire when we change from an `UnresolvedModel` -> Resolve model?

Reply:
Yes, we should investigate the underlying cause. The point of the system is that this happens automatically.

File a new issue. Start cataloguing places to update (e.g. this spot) once we work how to fix it. I wonder if we could use one of these new fangled signals to trigger the change.